### ISSUE TITLE
azuredevops_user table is not showing the correct value in column is_deleted_in_origin

### ISSUE THREAD

**Issue by jeffreymp17** [turbot/steampipe-plugin-azuredevops#23]
**Describe the bug**
The `azuredevops_user` table is not showing the correct value in the column `is_deleted_in_origin` is alwasy None

**Steampipe version (`steampipe -v`)**
v0.2.0

**Plugin version (`steampipe plugin list`)**
0.21.4

**To reproduce**

1. Delete a permanent user in Active Directory (Microsoft Entra)
2. Go back to execute the query to is the   `is_deleted_in_origin` is True 
3. After execute the query the value is None

**Expected behavior**
Show `is_deleted_in_origin` of the user that was deleted in the identity provider as True
**Additional context**

**Comment by ParthaI** [comment]
Hello @jeffreymp17,

Apologies for the delayed reply.

We are currently leveraging the [Graph::ListUsers API version 6](https://pkg.go.dev/github.com/microsoft/azure-devops-go-api/azuredevops/v6@v6.0.1/graph#ClientImpl.ListUsers) for populating the `azuredevops_user` table column. Unfortunately, we've encountered an issue where the API does not provide a value for the `IsDeletedInOrigin` property. Attempts to use version 7 of the API have also not yielded success.

To address this, we've submitted a support request to the Azure DevOps SDK repository, which can be followed here: [Issue #150](https://github.com/microsoft/azure-devops-go-api/issues/150). We plan to revisit and resolve this matter as soon as we receive feedback from the Azure DevOps SDK team.

Thank you for your patience and understanding.
